### PR TITLE
Use moist air R and cp in aerosol activation

### PR DESCRIPTION
**Pull request by trontrytel**
Before, the `c_p` and `R` in computation of maximum supersaturation in the `AerosolActivation` module were taken for the dry air. I think it's better to take them for moist air. The [original paper](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/97JD03735) was a little ambiguous with regards to that for me. I think they might have been also using the moist air constants somewhere. To double check, I found [this derivation](https://www.tandfonline.com/doi/abs/10.1080/00046973.1975.9648397).

This PR replaces eqs 11 and 12 from the [original paper](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/97JD03735) and dry air `R` and `c_p` with eq 10 from [here](https://www.tandfonline.com/doi/abs/10.1080/00046973.1975.9648397) using moist air `R` and `c_p`.

The results are basically unchanged. If you compare the example plot form the `AerosolActivation` documentation, the plot line shifts no more than its width. I still think that this is a more consistent approach.

This PR solves https://github.com/CliMA/CloudMicrophysics.jl/issues/28

### CODE REVIEW

File: docs/src/Aerosol_activation.md [hunk 1]
```diff
@@ -275,6 +280,14 @@ T = 294.0         # air temperature
 p = 1000.0 *1e2   # air pressure
 w = 0.5           # vertical velocity
 
+# We need the phase partition here only so that we can compute the
+# moist R and cp_m in aerosol activation module.
```
Comment (szy21) [inline docs/src/Aerosol_activation.md:284]:
```suggestion
# moist R_m and cp_m in aerosol activation module.
```

File: test/aerosol_activation_tests.jl [hunk 2]
```diff
@@ -16,6 +18,14 @@ T = 294.0       # air temperature K
 p = 100000.0    # air pressure Pa
 w = 0.5         # vertical velocity m/s
 
+# We need the phase partition here only so that we can compute the
+# moist air R and cp in aerosol activation module.
```
Comment (szy21) [inline test/aerosol_activation_tests.jl:22]:
```suggestion
# moist air R_m and cp_m in aerosol activation module.
```

**Review by szy21 — APPROVED** [review]
I trust your decision on using the moist air R_m and cp_m here:) Just left a minor comment on the docs.

File: test/aerosol_activation_tests.jl [hunk 3]
```diff
@@ -130,11 +140,11 @@ TT.@testset "callable and positive" begin
...
-        TT.@test AA.max_supersaturation(param_set, AM_t, T, p, w) > 0.0
-        TT.@test all(AA.N_activated_per_mode(param_set, AM_t, T, p, w) .> 0.0)
-        TT.@test all(AA.M_activated_per_mode(param_set, AM_t, T, p, w) .> 0.0)
-        TT.@test AA.total_N_activated(param_set, AM_t, T, p, w) > 0.0
-        TT.@test AA.total_M_activated(param_set, AM_t, T, p, w) > 0.0
+        TT.@test AA.max_supersaturation(param_set, AM_t, T, p, w, q) > 0.0
+        TT.@test all(AA.N_activated_per_mode(param_set, AM_t, T, p, w, q) .> 0.0)
+        TT.@test all(AA.M_activated_per_mode(param_set, AM_t, T, p, w, q) .> 0.0)
+        TT.@test AA.total_N_activated(param_set, AM_t, T, p, w, q) > 0.0
+        TT.@test AA.total_M_activated(param_set, AM_t, T, p, w, q) > 0.0
```
Comment (claresinger) [inline test/aerosol_activation_tests.jl:147]: it seems this isn't formatted correctly... apparently you can run the formatter locally and it should automatically correct this

**Review by claresinger — APPROVED** [review]
Looks good @trontrytel !

File: test/aerosol_activation_tests.jl (hunk 3 above)
Comment (trontrytel) [reply test/aerosol_activation_tests.jl:147]: done! (I always forget about it)

File: docs/src/Aerosol_activation.md (hunk 1 above)
Comment (trontrytel) [reply docs/src/Aerosol_activation.md:284]: done

File: test/aerosol_activation_tests.jl (hunk 2 above)
Comment (trontrytel) [reply test/aerosol_activation_tests.jl:22]: done

**Pull request merged**